recipes/deepsvr/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: deepsvr
+  version: "0.1.0"
+
+source:
+  url: https://github.com/griffithlab/DeepSVR/releases/download/v0.1.0/deepsvr-0.1.0.tar.gz
+  sha256: b95763c9fdfc718938ecf537a94714e2b4cbb1f083859faa7c86c2d90d22ef01 
+
+build:
+  noarch: python
+  script:
+    - flake8 . -v --config=.flake8rc
+    - nosetests --verbose
+    - python setup.py install
+  entry_points:
+    - deepsvr = deepsvr.cli:main
+
+requirements:
+  build:
+    - python==3.6.1
+    - setuptools
+    - nose
+    - flake8
+    - pandas==0.20.3
+    - numpy==1.12.1
+    - scikit-learn
+    - matplotlib
+    - seaborn
+    - bam-readcount
+    - convert_zero_one_based
+    - click
+    - tensorflow<=1.0.1
+    - keras==2.0.4
+    - h5py
+  run:
+    - python==3.6.1
+    - bam-readcount
+    - pandas==0.20.3
+    - numpy==1.12.1
+    - scikit-learn
+    - matplotlib
+    - seaborn
+    - convert_zero_one_based
+    - click
+    - tensorflow<=1.0.1
+    - keras==2.0.4
+    - scikit-learn
+    - h5py
+
+test:
+  requires:
+    - nose
```

Comment:
```yaml
imports:
  - deepsvr
commands:
  - deepsvr --version
```

Reply:
When I add a test of the commands I get the following error.

`SERR FileNotFoundError: [Errno 2] No such file or directory: 'locale'`

I am not sure why this is happening because it runs fine locally and on the local circleci.

For now I am going to remove the check. @corburn any ideas on how I can fix this?

Reply:
@bainscou based on the error, you may need to use the extended-base image #5842.

Reply:
That solved my issue! Thanks!